egs/babel_multilang/s5/local/nnet3/run_multilingual_bnf.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
+# This script trains a multilingual model using 6 layer TDNN + Xent
+# with 42 dim bottleneck layer in th fifth layer.
+# Then it extracts bottleneck features for input language "lang" and
+# train SAT model using these feautures.
+
+# Copyright 2016  Pegah Ghahremani
+# Apache 2.0
+
+set -e           #Exit on non-zero return code from any command
+set -o pipefail  #Exit if any of the commands in the pipeline will
+                 #return non-zero return code
+. conf/common_vars.sh || exit 1;
+
+set -u           #Fail on an undefined variable
+bnf_train_stage=-10 # the stage variable used in multilingual bottleneck training.
+stage=1
+speed_perturb=true
+multilingual_dir=exp/nnet3/multi_bnf
+global_extractor=exp/multi/nnet3/extractor
+lang_list=(404-georgian 403-dholuo 402-javanese 401-mongolian 307-amharic)
```

Comment:
should not lang_list and perhaps other values be obtained from local.conf?
also, ad lang_list -- arrays are quite complicated to be passed as parameters and parse_options.sh do not really support (AFAIK), so the value is effectively hardcoded.